srcmedia/ts/lib/output.ts

```diff
@@ -0,0 +1,27 @@
+import { Subject } from 'rxjs'
+
+import { Component, Reactive } from '../lib/common'
+import { ImageLazyLoader } from '../../js/modules/LazyLoad'
+
+type RxOutputState = string
+
```

Comment:
Missing docstring here to document what this is. This is for updating the results?

Reply:
yeah, for now it's just a "dumb" representation of the returned html as a string. I'll add a note